core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -4174,11 +4244,14 @@ class ReplicaManagerTest {
 class MockReplicaSelector extends ReplicaSelector {
 
   private val selectionCount = new AtomicLong()
+  private var partitionViewArgument: Option[PartitionView] = None
 
   def getSelectionCount: Long = selectionCount.get
+  def getPartitionViewArgument(): Option[PartitionView] = partitionViewArgument
```

Comment:
nit: We usually don't prefix getters with `get`. Could we rename these?

Reply:
i followed the convention used with `getSelectionCount` due to
```
partitionViewArgument is already defined in the scope
```